keyboards/keychron/c1_pro/ansi/white/config.h

```diff
@@ -0,0 +1,74 @@
+
+/* Scan phase of led driver set as MSKPHASE_9CHANNEL(defined as 0x03 in CKLED2001.h) */
+#define PHASE_CHANNEL MSKPHASE_9CHANNEL
+/* Set led driver current */
+#define CKLED2001_CURRENT_TUNE \
+    { 0x9D, 0x9D, 0x44, 0x9D, 0x9D, 0x44, 0x9D, 0x9D, 0x44, 0x9D, 0x9D, 0x44 }
+
+/* turn off effects when suspended */
+#define LED_DISABLE_WHEN_USB_SUSPENDED
+
+/* Enbale caps_lcok, win os and mac os indicator */
+#define CAPS_LOCK_LED_INDEX 63
+#define MAC_LOCK_LED_INDEX 64
+#define WIN_LOCK_LED_INDEX 65
+
+// LED Matrix Animation modes. Explicitly enabled
+// For full list of effects, see:
+// https://docs.qmk.fm/#/feature_led_matrix?id=led-matrix-effects
+#define ENABLE_LED_MATRIX_NONE
+#define ENABLE_LED_MATRIX_SOLID // Static single val, no speed support
+// #define ENABLE_LED_MATRIX_ALPHAS_MODS              // Static dual val, speed is val for LEDs marked as modifiers
+#define ENABLE_LED_MATRIX_BREATHING // Cycling brightness animation
+// #define ENABLE_LED_MATRIX_BAND                     // Band fading brightness scrolling left to right
+#define ENABLE_LED_MATRIX_BAND_PINWHEEL    // 3 blade spinning pinwheel fades brightness
+#define ENABLE_LED_MATRIX_BAND_SPIRAL      // Spinning spiral fades brightness
+#define ENABLE_LED_MATRIX_CYCLE_LEFT_RIGHT // Full gradient scrolling left to right
+#define ENABLE_LED_MATRIX_CYCLE_UP_DOWN    // Full gradient scrolling top to bottom
+#define ENABLE_LED_MATRIX_CYCLE_OUT_IN     // Full gradient scrolling out to in
+#define ENABLE_LED_MATRIX_DUAL_BEACON      // Full gradient spinning around center of keyboard
+// #if defined(LED_MATRIX_KEYPRESSES) || defined(LED_MATRIX_KEYRELEASES)
+#define LED_MATRIX_KEYPRESSES
+#define LED_MATRIX_KEYRELEASES
+
+#define ENABLE_LED_MATRIX_SOLID_REACTIVE_SIMPLE // Pulses keys hit then fades out
+// #define ENABLE_LED_MATRIX_SOLID_REACTIVE_WIDE       // Value pulses near a single key hit then fades out
+#define ENABLE_LED_MATRIX_SOLID_REACTIVE_MULTIWIDE // Value pulses near multiple key hits then fades out
+#define ENABLE_LED_MATRIX_SOLID_REACTIVE_CROSS     // Value pulses the same column and row of a single key hit then fades out
+// #define ENABLE_LED_MATRIX_SOLID_REACTIVE_MULTICROSS // Value pulses the same column and row of multiple key hits then fades out
+// #define ENABLE_LED_MATRIX_SOLID_REACTIVE_NEXUS      // Value pulses away on the same column and row of a single key hit then fades out
+// #define ENABLE_LED_MATRIX_SOLID_REACTIVE_MULTINEXUS // Value pulses away on the same column and row of multiple key hits then fades out
+#define ENABLE_LED_MATRIX_SOLID_SPLASH      // Value pulses away from a single key hit then fades out
+#define ENABLE_LED_MATRIX_SOLID_MULTISPLASH // Value pulses away from multiple key hits then fades out
+// #endif
+#define ENABLE_LED_MATRIX_WAVE_LEFT_RIGHT // Sine wave scrolling from left to right
+#define ENBALE_LED_MATRIX_WAVE_UP_DOWN    // Sine wave scrolling from up to down
+// #define ENABLE_LED_MATRIX_EFFECT_MAX
```

Comment:
```suggestion
// #if defined(LED_MATRIX_KEYPRESSES) || defined(LED_MATRIX_KEYRELEASES)
#define LED_MATRIX_KEYPRESSES
#define LED_MATRIX_KEYRELEASES
```
Same suggestion as for the `rgb` variant.